tensorflow/tools/ci_build/mac/gpu/pip/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+#
+# Usage:
+#   This script assumes it is invoked while `pwd` points to the repository root.
+#   ./tensorflow/tools/ci_build/mac/gpu/pip/run.sh
+#
+#   By default, script will use python2.
+#   You can elect to use python 3 by setting TF_PYTHON_VERSION environment
+#   variable to "3".
+#
+
+# Mac seems to be losing the environment variables, remind the important ones:
+export PATH="/usr/local/cuda/bin:/usr/local/cuda/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
+export LB_LIBRARY_PATH="/usr/local/cuda/lib:/usr/local/cuda/extras/CUPTI/lib:/usr/local/cuda/lib:/usr/local/cuda/extras/CUPTI/lib:"
+
+# Enable stop on errors, and debugging.
+set -e
+set -x
+
+# This script is under <repo_root>/tensorflow/tools/ci_build/mac/gpu/pip/
+# Save repository root directory for future use.
+REPO_ROOT=`pwd`
```

Comment:
That seems to behave differently under different terminals offered by mac, linux and msys (on windows), That's why I elected to go for the simpler approach.
Also, do we need that much robustness in this script?
It will immediately fail when trying to source builds_common.sh if this does not work correctly.